docker/Dockerfile.linux.amd64

```diff
@@ -5,7 +5,8 @@ LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
   org.label-schema.vendor="Drone.IO Community" \
   org.label-schema.schema-version="1.0"
 
-RUN apk add -U ca-certificates python3 && \
```

Comment:
Dev packages should be added in sepperate cmd removed after pip3 install when not needed anymore

Reply:
Move these packages `python3-dev gcc libc-dev libffi-dev openssl-dev` to after pip3 install?

Reply:
I ment something like this:
```
RUN apk add -U --no-cache ca-certificates python3
RUN apk add -U --no-cache --virtual build-dependencies python3-dev gcc libc-dev libffi-dev openssl-dev && \
    pip3 install --no-cache-dir --upgrade pip setuptools wheel six twine && \
    apk del build-dependencies
```

not tested but imho should work. This would additional layer but would make final image much smaller